src/types/signed_price.ts

```diff
@@ -0,0 +1,17 @@
+import { BigNumber } from "../../utils/bignumber";
+
+import { ECDSASignature } from "./ecdsa_signature";
+
+/**
+ * A price that has been signed by some price feed.
+ */
+interface SignedPrice {
+    // The symbol of the token being priced, e.g. "REP".
+    tokenSymbol: string;
```

Comment:
I believe this should be tokenAddress; we can add the tokenSymbol at the Price class layer, but the blockchain should confirm the tokenAddress, I think